airbyte-integrations/connectors/source-bigquery/source_bigquery/schema_helpers.py

```diff
@@ -0,0 +1,103 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+import logging
+from copy import deepcopy
+from typing import Any, Dict
+
+from airbyte_cdk.models import AirbyteStream
+from airbyte_cdk.models.airbyte_protocol import DestinationSyncMode, SyncMode
+
+logger: logging.Logger = logging.getLogger("airbyte")
+
+
+class SchemaTypes:
+
+    string: Dict = {"type": ["null", "string"]}
+
+    number: Dict = {"type": ["null", "number"]}
+
+    boolean: Dict = {"type": ["null", "boolean"]}
+
+    array: Dict = {"type": ["null", "array"]}
+
+    object: Dict = {"type": ["null", "object"]}
+
+
+# https://docs.airbyte.com/integrations/sources/bigquery
+SIMPLE_BIGQUERY_TYPES: Dict = {
+    "BOOL": SchemaTypes.boolean,
+    "INT64": SchemaTypes.number,
+    "FLOAT64": SchemaTypes.number,
+    "NUMERIC": SchemaTypes.number,
+    "BIGNUMERIC": SchemaTypes.number,
+    "INTEGER": SchemaTypes.number,
+    "STRING": SchemaTypes.string,
+    "BYTES": SchemaTypes.string,
+    "DATE": SchemaTypes.string,
+    "DATETIME": SchemaTypes.string,
+    "TIMESTAMP": SchemaTypes.string,
+    "TIME": SchemaTypes.string,
+    "GEOGRAPHY": SchemaTypes.string
+}
+
+COMPLEX_BIGQUERY_TYPES: Dict = {
+    "ARRAY": SchemaTypes.array,
+    "RECORD": SchemaTypes.array,
```

Comment:
Based on [this response](https://stackoverflow.com/a/62495932/2800559), I would assume RECORD and STRUCT to have the same type. Can you explain why this is handled differently?

Reply:
Fixed here
https://github.com/artefactory/airbyte/blob/0437ac48aee0e20b2542131f7e30048f61099e73/airbyte-integrations/connectors/source-bigquery/source_bigquery/schema_helpers.py#L47